simulator/func_sim/operation.h

```diff
@@ -66,15 +67,19 @@ class Operation
 	//target is known at ID stage and always taken
 	bool is_direct_jump() const { return operation == OUT_J_JUMP; }
 
+	//target is known at ID stage; likely to be taken
+	bool is_likely_branch() const { return operation == OUT_BRANCH_LIKELY; }
+
 	//target is known at ID stage but if branch is taken or not is known only at EXE stage
 	bool is_branch() const { return operation == OUT_BRANCH; }
```

Comment:
Do you agree that a likely branch is a branch?

Reply:
Yes. I add OUT_BRANCH_LIKELY in order to differ them from usual branches.

Reply:
Yep, but current naming is misleading. I suggest `is_branch` be true for likely and common branches. You may introduce another method `is_common_branch`.